ios/Classes/FlutterLocalNotificationsPlugin.m

```diff
@@ -399,7 +404,7 @@ - (void) showLocalNotification:(NotificationDetails *) notificationDetails {
     if(notificationDetails.secondsSinceEpoch == nil) {
         if(notificationDetails.repeatInterval != nil) {
             NSTimeInterval timeInterval = 0;
-            
+            NSInteger dayInterval = notificationDetails.dayInterval == nil ? 1 : [notificationDetails.dayInterval integerValue];
```

Comment:
similar comments to what I made about the Android change. Just a note that the issue I mentioned was more of a concern on Android where the plugin needs to handle rescheduling notifications when the phone is restarted